packages/react-components/theme-designer/src/components/ExportButton/ExportButton.tsx

```diff
@@ -44,7 +27,7 @@ export const ExportButton: React.FC<ExportProps> = props => {
         <MenuPopover>
           <MenuList>
             <MenuItem>
-              <ExportLink brand={brand} isDark={isDark} overrides={overrides} />
+              <ExportLink />
```

Comment:
I think we're going to need to look at this in an a11y review, since a link can't be nested inside a menuitem 😅

(not something to change in this PR though)

Reply:
Maybe we can create a separate work item to track this and not forget it?

Reply:
i will work on this as soon as this PR gets merged :)